src/proto/plumtree.rs

```diff
@@ -546,9 +546,11 @@ impl<PI: PeerIdentity> State<PI> {
                         id: None,
                         round: ihave_round,
                     });
+                    self.add_eager(ihave_peer);
                     io.push(OutEvent::SendMessage(ihave_peer, message));
                 }
                 // Prune the sender of the Gossip.
+                self.add_lazy(*gossip_sender);
```

Comment:
random line on this commit (`fix(plumtree): ensure eager relation is synchronous`) do you mean symmetrical?